Split/Storage/MySegments/MySegmentsStorage.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  MySegmentsStorage.swift
+//  Split
+//
+//  Created by Javier L. Avrudsky on 09/11/2020.
+//  Copyright © 2020 Split. All rights reserved.
+//
+
+import Foundation
+
+protocol MySegmentsStorage {
+    func loadLocal()
```

Comment:
Maybe it is considered in another PR, but MySegmentsStorage is missing the method `belongsTo` that given a segment name returns whether it belongs to the set of segments or not.

Reply:
Hey Doc, not sure if I understand why is missing. Let's discuss in Slack.